filesystem/usr/local/share/landoop/etc/message

```diff
@@ -1,5 +1,6 @@
 \e[36m********************************************************************************
-\e[36mYou may visit \e[96mhttp://${PRINT_HOST}:${WEB_PORT}\e[36m in \e[96mabout 45 seconds\e[36m. Login with \e[96madmin/admin\e[36m.
+\e[36mYou may visit \e[96mhttp://${PRINT_HOST}:${WEB_PORT}\e[36m in \e[96mabout 45 seconds\e[36m. Login with \e[96madmin/admin\e[36m
+\e[36mfor full access or logviewer/logviewer for read-only access.
```

Comment:
Please revert this change. We do not want to show it here. :)

Reply:
Done